.github/workflows/ci.yml

```diff
@@ -320,6 +320,64 @@ jobs:
         # reason for excluding https://github.com/napi-rs/napi-rs/issues/2200
         run: cargo test --workspace --exclude rspack_binding_options --exclude rspack_node -- --nocapture
 
+  rust_test_codecov:
+    name: Rust Test Coverage
+    needs: [rust_changes]
+    if: ${{ needs.rust_changes.outputs.changed == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Pnpm Cache # Required by some tests
+        uses: ./.github/actions/pnpm-cache
+
+      - name: Install Rust Toolchain
+        uses: ./.github/actions/rustup
+        with:
+          save-cache: ${{ github.ref_name == 'main' }}
+          shared-key: check
+
+      - name: Install llvm-tools-preview
+        run: rustup component add llvm-tools-preview
+      - uses: cargo-bins/cargo-binstall@v1.10.2
+      - run: cargo binstall --no-confirm cargo-llvm-cov
+
+      - name: Run test coverage
+        # reason for excluding https://github.com/napi-rs/napi-rs/issues/2200
+        run: cargo llvm-cov --workspace --exclude rspack_binding_options --exclude rspack_node --lcov --output-path lcov.info
```

Comment:
Why do we need to run test in two different jobs (rust_test and rust_test_codecov), can we unify them?

Reply:
done, merged in `rust_test` with `cargo test`